app/Lib/BulkUploader.php

```diff
@@ -470,17 +471,14 @@ public static function processTickets($records, $action, $commit, $reason): void
              * 2020 is year 3 ... and it expires AFTER the 2020 event.
              */
 
-            if ($type == "CRED" || $type == "RPT" || $type == "GIFT") {
```

Comment:
@mikeburg please confirm whether it's correct to remove GIFT from this check. It's my understanding from the Issue that gift tickets shouldn't get the year+3 expiry date, and also that they shouldn't allow for someone to set sourceYear or expiryYear for them from the bulk uploader.

Please confirm this either way

Reply:
@techxorcist I'm curious if you agree with this too

Reply:
@srabraham No, don't remove GIFT tickets. The plan is to implement GIFT & LSD tickets via https://github.com/burningmantech/ranger-clubhouse-web/issues/1121

Reply:
Done. I'm not actually sure what gift tickets are then, if they're not LSD tickets